templates/usr/local/sbin/secure_owncloud.sh.j2

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# This script sets the recommended modes and ownership for your ownCloud 
+# directories and files
+# https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label
+
+# {{ ansible_managed }}
+
```

Comment:
@drybjed I just rechecked and found this:
```Shell
set -o nounset -o pipefail -o errexit
```

in  [Defensive BASH programming](https://news.ycombinator.com/item?id=10736584) which I think is more readable and which I would propose to use instead. Thumbs up or down?